src/clib/lib/include/ert/job_queue/job_status.hpp

```diff
@@ -90,7 +90,6 @@ typedef enum {
     JOB_QUEUE_RUNNING_EXIT_CALLBACK = 2048,
     JOB_QUEUE_STATUS_FAILURE = 4096,
     JOB_QUEUE_FAILED = 8192,
-    JOB_QUEUE_DO_KILL_NODE_FAILURE = 16384,
```

Comment:
Reading the inline docs above: `NB: the status count algorithm has a HARD assumption that these
  values are on the 2^N form - without holes in the series. ` this should fail HARD.

Reply:
Will update the value then